data/data/manifests/openshift/samples-operator.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: samples.operator.openshift.io/v1
+kind: Config
+metadata:
+  name: cluster
+spec:
+  architectures:
+    - x86_64
```

Comment:
without arch:
"'cannot change architectures from []string{"x86_64"} to []string(nil)'"

Reply:
Good catch. I suppose its a samples operator bug probably, but this solution is temporary anyway, lets roll with it